src/infra/awsS3/aws.service.ts

```
@@ -2,11 +2,15 @@ import { Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
 
 import { PutObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { VALID_IMAGE_EXTENSIONS } from './aws.valid-extensions';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)